styles.css

```diff
@@ -0,0 +1,391 @@
+    font-weight: 400;
+}
+
+div img {
+    max-width: 100%;
+}
+
+article {
+    clear: both;
+    padding: 25px;
+}
+
+article img {
+    padding: 0 25px 25px 0;
+    float: left;
+}
+
+article h3 {
+    font-family: 'Arbutus Slab', serif;
+    color: #093C60;
+    font-size: 1.5rem;
+    display: inline;
+    }
+
+article p {
+    font-family: 'Work Sans', sans-serif;
+    font-size: 1rem;
+}
+
+    footer {
+        font-family: 'Work Sans', sans-serif;
+        clear: both;
+        text-align: center;
+        width: 100%;
+        height:20%;
+        background-color: #3D3D3D;
+        color:white;
+        padding: 5% 13%;
+        margin-top: 100px;
+        line-height: 2rem;
+    }
+
+    footer img {
+        max-width: 35%;
+        min-width: 20rem;
+        height: auto;
+        padding-top: 2rem;
+    }
+    
+    footer nav {
+        background-color: #3D3D3D;
+        font-weight: 700;
+    }
+    
+    footer nav a {
+        color: white;
+        text-decoration: none;
+        font-size: 16px;
+    }
+    
+    footer nav a:visited {
+        text-decoration: none;
+    }
+    
+    footer p {
+        font-weight: 400;
+        font-size: 16px;
+    }
+
+
+.column {
+    float: left;
+    width: 25%;
+    padding: 15px 10px;
+  }
+  
+  .row {
+    margin: 0 -5px;
+}
+  
+  .row:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  .card {
+    padding: 0px;
+    text-align: left;
+    border: #EDEDED;
+    border-style: solid;
+    border-width: 2px;
+  }
+
+  div h3 {
+    font-family: 'Arbutus Slab', serif;
+    color: #093C60;
+    font-size: 1.2rem;
+    margin-left: 10px;
+    margin-bottom: -5px;
+  }
+
+  div h4 {
+    font-family: 'Work Sans', sans-serif;
+    font-size: 1rem;
+    font-weight: 700;
+    margin-left: 10px;
+    margin-bottom: -5px;
+  }
+
+  div p {
+    font-family: 'Work Sans', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    margin-left: 10px;
+    margin-bottom: -5px;
+  }
+
+  .card-button {
+    background-color: #81B741;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 700;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    width: 90%;
+    border-radius: 2px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    display: block !important;
+  }
+
+  .see-more-button {
+    background-color: #ECECEC;
+    border: none;
+    color: #093C60;
+    padding: 10px 20px;
+    text-align: center;
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 700;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 20px;
+    width: 23%;
+    border-radius: 2px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    display: block !important;
+  }
+
+ /* Mobile Styles */
+ 
+  @media screen and (max-width: 800px) {
```

Comment:
We'll come back to this later on in the course., but a great idea to include it now. Some of the homework later on will be a bit easier.